zipline/data/hdf5_daily_bars.py

```diff
@@ -602,6 +690,40 @@ def asset_start_dates(self):
     def asset_end_dates(self):
         return self.dates[self._country_group[LIFETIMES][END_DATE][:]]
 
+    @lazyval
+    def _currency_codes(self):
+        return self._country_group[CURRENCY][CODE][:]
+
+    def currency_codes(self, sids):
```

Comment:
Mainly just for my understanding of the code: where is this `self.currency_codes()` being used (or going to be used)?

Reply:
It's not currently being used by any of the code on this branch. I'm working on updating the equity pricing pipeline loader (https://github.com/quantopian/zipline/blob/master/zipline/pipeline/loaders/equity_pricing_loader.py#L39-L69) to be able to read this data when it's requested.